### PR TITLE
Update to Android v2 embedding

### PR DESCRIPTION
**Pull request by DRSchlaubi**
This PR upgrade to android v2 embedding you should probabbly provide documentation on v1 embedding no longer being supported

## Changes
- Update example app - Target Android 11 - Bump gradle to 6.6.1 - Migrate to v2 embedding
- Migrate plugin to android v2 - update kotlin to 1.4.30 - Update to latest android - Use new v2 embedding

P.S.: Please add the `hacktoberfest` label to here I want this to count :D thx

### CODE REVIEW

File: example/android/gradle.properties [hunk 1]
```diff
@@ -1 +1,3 @@
-org.gradle.jvmargs=-Xmx1536M
+org.gradle.jvmargs=-Xmx1536M[
```
Comment (matteocrippa) [inline example/android/gradle.properties]: The extra [ is a typo?

**Comment by DRSchlaubi** [comment]
The label is actually called "hacktoberfest-accepted" my bad

File: example/android/gradle.properties (hunk 1 above)
Comment (matteocrippa) [reply example/android/gradle.properties]: @DRSchlaubi ?

**Comment by matteocrippa** [comment]
@DRSchlaubi you have pending review stuff to check

**Comment by DRSchlaubi** [comment]
Sorry what do you mean?

**Comment by matteocrippa** [comment]
2020-10-03-180950_shot

**Comment by DRSchlaubi** [comment]
For some reason github does not display that to me

**Comment by DRSchlaubi** [comment]
Github still not display your review to me, is there anything else than the typo?
and yes it's a typo didn't do that intentionally

**Comment by DRSchlaubi** [comment]
So the typo should be fixed now

**Comment by matteocrippa** [comment]
@DRSchlaubi thx for PR

**Pull request merged**